### ISSUE TITLE
handleSyncClick is not defined on the onClick prop

### ISSUE THREAD

**Issue by MooreDevelopment** [abdennour/react-csv#96]
Issue with PR #66 
I used the onClick example from the Sync+Proceed section. When running it i am getting "handleSyncClick is not defined" when i try to use it. 

I have the component set as such.

```js
prop ={
                            className={`button`}
                            filename={`Results_${moment().format('YYYY-MM-DD_hh:mm:ss:SS')}.csv`} 
                            data={this.state.data}
                            onClick={()=>{
                                console.log('You Clicked the link')
                            }}
                            headers={[
                                { key: 'DATA1', label: 'Data 1' }, 
                                { key: 'DATA2', label: 'Data 2' }
                            ]}
}
```

if you need any more clarification don't hesitate to reach out!

**Comment by abdennour** [comment]
Duplicated of #98

**Comment by abdennour** [comment]
Fixed since release **1.0.15-RC2**
